### PR TITLE
Add dashflo.net/deploys.io

### PR DESCRIPTION
**Pull request by itslukej**
Adds deploys.io

### CODE REVIEW

**Comment by Daxorinator** [comment]
No proof, terrible reasons for listing, company is registered.

Please post something worth our time reading besides shitposting.
I'm not happy with deploys either, but they're not a scam.

Motion to close.

**Comment by itslukej** [comment]
> Company is registered

Doesn't mean anything. There are other businesses on the list that are registered such as MCSHosting Ltd.
> No proof

https://www.trustpilot.com/review/deploys.io?languages=en&stars=1
> Terrible reasons for listing

Feel free to edit.

**Comment by FrozenPvPs** [comment]
Just because the uptime isn't that good doesn't mean that they are a SummerHost.
15-16 year old staff are perfectly fine.
I do agree with the lack of transparency though.

**Comment by koko-0012** [comment]
didn't the host have problems cause of server had been hit by something
for few days tell everyone what happen the host was fine before august 
and reason for being down was good reason :?
So why would say it's summerhoax for stupid thing that no knew that would happen.

**Comment by RhvsW** [comment]
Not much of a summer host list if this doesn't get added.

**Comment by Daxorinator** [comment]
Once again, no proof in the PR other than your link to TrustPilot, most reviews of which are about downtime. There's nothing wrong with having young staff, I know most of them personally and they're well qualified. Shit happens, downtime happens, deal with it, move on.

As Koko half-arsedly said, they had issues that plagued their network that wasn't their fault.

Nobody wants to edit this half-arsed PR either. If you want to get something accepted, write up a proper pull request, with proper evidence and reasoning as to why the host in question should be added, then submit it and ask for feedback.

**Comment by itslukej** [comment]
> Just because the uptime isn't that good doesn't mean that they are a SummerHost.

Is this not inferred in Step 1? (Buying cheap servers)
> 15-16 year old staff are perfectly fine.

No.
> I do agree with the lack of transparency though.

👍 
> didn't the host have problems cause of server had been hit by something
> for few days tell everyone what happen the host was fine before august
> and reason for being down was good reason :?
> So why would say it's summerhoax for stupid thing that no knew that would happen.

Deploys.io is liable for their downtime.

**Comment by RhvsW** [comment]
> 15-16 year old staff are perfectly fine.

No they're really not.

**Comment by Daxorinator** [comment]
Who says they buy cheap servers? They actually own most of their hardware and it sure as fuck isn't cheap. Like we said already, it's networking issues that cause the outages.
The 17 year old staff are perfectly fine, they are qualified and didn't have issues like this before until their upstream provider started being stupid.

They are liable for their downtime, yes, and to my knowledge are compensating their clients for the downtime.

**Comment by RhvsW** [comment]
> they are qualified

{{ citation needed }}



> They actually own most of their hardware

> it sure as fuck isn't cheap

1. buy hardware off ebay
2. make money renting it
3. shut down
4. sell hardware on ebay for similar price or higher
5. ???

**Comment by koko-0012** [comment]
you know if they did that i wonder why i havent seen some report something else but
Downtime. 
If they did that they would not have such powerful systems

**Comment by DamonOnYT** [comment]
Whoever made this PR has -2 IQ

**Comment by Dot-Rar** [comment]
Guess I'll bite the bait and dive in.

>Who says they buy cheap servers? They actually own most of their hardware and it sure as fuck isn't cheap
€6.43/U _including_ Voxility is super, super, super cheaper. There's no surprises that there's network issues.

>The 17 year old staff are perfectly fine, they are qualified
Which qualifications do they hold?

>didn't have issues like this before until their upstream provider started being stupid.
They switched to Combahton, saw that Combahton wasn't really stopping attacks. Stuck with Combahton. Then the Combahton complete network blackout occurred, and they announced that they'd be dropping Combahton on a specific date. Never happened, still with Combahton.

>and to my knowledge are compensating their clients for the downtime.
Some have, but very few. Pretty sure <5 were compensated during that Combahton network outage.

>you know if they did that i wonder why i havent seen some report something else but
Downtime.
Alright, remember when they had a total data breach (after GDPR in btw)? As someone who is actually qualified to deal with incidents and breaches, the way they handled it makes me cringe. Here's a tldr of what went down, iirc:

- Guy got Tim's passwords, reused creds to get into the backup server
- Dumped database, got creds and logged into a WHMCS admin account
- From there he had complete access to anything, which would include card details if deploys.io haven't explicitly disabled WHMCS's option to store card details (enabled by default).

Their response?
image
Yeah, that's it. Didn't report to the ICO or anything.

There's just a couple things I'd like to point out:
>no passwords, bank or card details have been accessed
From this I'd assume that deploys.io have disabled storing card details, but passwords? Well, WHMCS exposes the root passwords of every VPS to admins. Customers were not made aware of this or forced to change passwords.

>an individual with non-malicious intent
Well, while deploys.io were still figuring out the breach, the guy socially engineered a customer with a dedicated server and installed a backdoor + BTC miner. Sounds pretty malicious to me.

>but did not export any user data
They don't know that. They didn't perform any serious forensic analysis (didn't even take a disk image), or perform the containment and lessons learned stages of incident handling.

**Comment by itslukej** [comment]
> From there he had complete access to anything, which would include card details if deploys.io haven't explicitly disabled WHMCS's option to store card details (enabled by default).

yikes

**Comment by Ethereal739** [comment]
> Guess I'll bite the bait and dive in.
> 
> > Who says they buy cheap servers? They actually own most of their hardware and it sure as fuck isn't cheap
> 
> €6.43/U _including_ Voxility is super, super, super cheaper. There's no surprises that there's network issues.
> 
> > The 17 year old staff are perfectly fine, they are qualified
> 
> Which qualifications do they hold?
> 
> > didn't have issues like this before until their upstream provider started being stupid.
> 
> They switched to Combahton, saw that Combahton wasn't really stopping attacks. Stuck with Combahton. Then the Combahton complete network blackout occurred, and they announced that they'd be dropping Combahton on a specific date. Never happened, still with Combahton.
> 
> > and to my knowledge are compensating their clients for the downtime.
> 
> Some have, but very few. Pretty sure <5 were compensated during that Combahton network outage.
> 
> > you know if they did that i wonder why i havent seen some report something else but
> > Downtime.
> 
> Alright, remember when they had a total data breach (after GDPR in btw)? As someone who is actually qualified to deal with incidents and breaches, the way they handled it makes me cringe. Here's a tldr of what went down, iirc:
> 
> * Guy got Tim's passwords, reused creds to get into the backup server
> * Dumped database, got creds and logged into a WHMCS admin account
> * From there he had complete access to anything, which would include card details if deploys.io haven't explicitly disabled WHMCS's option to store card details (enabled by default).
> 
> Their response?
> image
> Yeah, that's it. Didn't report to the ICO or anything.
> 
> There's just a couple things I'd like to point out:
> 
> > no passwords, bank or card details have been accessed
> 
> From this I'd assume that deploys.io have disabled storing card details, but passwords? Well, WHMCS exposes the root passwords of every VPS to admins. Customers were not made aware of this or forced to change passwords.
> 
> > an individual with non-malicious intent
> 
> Well, while deploys.io were still figuring out the breach, the guy socially engineered a customer with a dedicated server and installed a backdoor + BTC miner. Sounds pretty malicious to me.
> 
> > but did not export any user data
> 
> They don't know that. They didn't perform any serious forensic analysis (didn't even take a disk image), or perform the containment and lessons learned stages of incident handling to my knowledge.

When did this happen and aren't you under NDA?

**Comment by Dot-Rar** [comment]
@Ethereal739 data breach occurred late January, and nope, I haven't signed anything at all

**Comment by nikrozman21** [comment]
It really sucks for them as they are (were?) pretty decent.

The network blackouts can't be fully offloaded to the datacenter as I'm sure it isn't DC's networking equipment that failed, but rather the customer's - deploys.io's. Had it been taken care of sooner or in case they had some form of redundancy this likely would have been fixed days earlier. There are other factors to count in here such as the price of new equipment, failure severity, their provider's flexibility on the matter, etc.

As for the data breach, if all clients were instructed to follow certain security procedures to protect themselves from future complications, they did well however it definitely would be better if they haven't had a breach in the first place.

All other bullshit you guys pulled up in this thread is mostly stuff you shouldn't really care enough cause I am pretty sure a bunch other hosts including the one I work at had issues that could potentially get us all put up on SummerHoax, but that never happens cause the severity of the issues isn't that high or is completely irrelevant to the end-user's security or to their use of services or either just quality.

**Comment by Dot-Rar** [comment]
>I'm sure it isn't DC's networking equipment that failed

To be fair, the Combahton outage was caused by Combahton's maintenance on a core router gone wrong iirc. They also used to use tower colocation (servers are racked as of a month or two ago), where they have absolutely no control over the network gear at all. Sure, you could still criticise them for choosing poor providers and whatnot, but many of the faults haven't been caused by them directly.

**Comment by nikrozman21** [comment]
I can assure you, it wasn't Combahton's directly.

**Comment by Daxorinator** [comment]
Sorry Nik, gotta agree with Dot-Rar on this one, I can swear on my great grandfathers grave, it was Combahton (or some other upstream? I'm not great with the names) that was at fault here. For moral reasons I won't elaborate but I know for a fact it's the upstream's fault and that deploys networking equipment wasn't the reason for total system failure.

**Comment by Dot-Rar** [comment]
>I can assure you, it wasn't Combahton's directly.

https://status.combahton.net/incident/129

**Comment by itslukej** [comment]
Combahton's fault or not it doesn't change that Deploys was liable for the downtime they caused their customers.

**Comment by Ethereal739** [comment]
> Combahton's fault or not it doesn't change that Deploys was liable for the downtime they caused their customers.

They did issue compensation though?

**Comment by itslukej** [comment]
> They did issue compensation though?

From Dot-Rar's post:
> > and to my knowledge are compensating their clients for the downtime.
> 
> Some have, but very few. Pretty sure <5 were compensated during that Combahton network outage.

**Comment by Ethereal739** [comment]
> > They did issue compensation though?
> 
> From Dot-Rar's post:
> 
> > > and to my knowledge are compensating their clients for the downtime.
> > 
> > 
> > Some have, but very few. Pretty sure <5 were compensated during that Combahton network outage.

He literally just crossed that out.

**Comment by TheDeathStreamer** [comment]
I am great friends with deploys and I have never had any trouble with him, he is a great guy no lie.

**Comment by RhvsW** [comment]
> I am great friends with deploys and I have never had any trouble with him, he is a great guy no lie.

Yes officer he is my friend he would never speed trust me, great guy no lie.

**Comment by TheDeathStreamer** [comment]
> > I am great friends with deploys and I have never had any trouble with him, he is a great guy no lie.
> 
> Yes officer he is my friend he would never speed trust me, great guy no lie.

He really isnt I wasnt saying as a joke, I may have never owned one of his products but I know he wouldnt deliever horriable products. He may have friends on summerhoax but with this guy providing 1. Absolutely nothing that dxploys has done to him no reason on why he should be on here is the fact that he just wants the dude on there cause he may not like dxploys or su m .

**Comment by Daxorinator** [comment]
"I am great friend with deploys I have never had any trouble with him"

I think this statement speaks for itself lmao...
Where's this deploys guy can I meet him?

**Comment by Ethereal739** [comment]
So what I am getting here is:

1. Deploys has a lot of downtimes -> Yes, and they did issue compensations for that.
2. Deploys has bad communication -> Yes, that is clearly what they need to fix. However, this is not a reason to add them to the list.
3. Deploys got a breach -> They did email the clients regarding this issue and recommended that all passwords should be changed. 
4. They are teenagers -> Yes, and they did legally register the company using their names. We do not list people here because of age (of course unless they are little kids who steal their parents credit cards and whatnot). MCSHosting is owned by NickE, who literally attacked a competitor to steal customers (and surprise surprise, that competitor was DeluxeNode which later became Deploys.io).

image

image

image


While I do agree that Deploys have had a ton of issues within the last month, it is important to note that we only list scammers and hosts who get involved in shady/illegal acts (such as MCSHosting).

So yes, Deploys.io's service has been quite frankly bad since the outage. However, I do not see them lying/scamming/committing crimes. This is purely a case of low uptime and horrible support. 

I will not accept this PR unless there are new developments in the next few days.

**Comment by Nobody196** [comment]
The breach was never reported?

**Comment by TheDeathStreamer** [comment]
> "I am great friend with deploys I have never had any trouble with him"
> 
> I think this statement speaks for itself lmao...
> Where's this deploys guy can I meet him?

Dxploys lost his phone fishing and i've been talking to him on discord lately was talking to him yesterday.

**Comment by Ethereal739** [comment]
> > "I am great friend with deploys I have never had any trouble with him"
> > I think this statement speaks for itself lmao...
> > Where's this deploys guy can I meet him?
> 
> Dxploys lost his phone fishing and i've been talking to him on discord lately was talking to him yesterday.

Deploys is not even a person and if you are only here to for the sole purpose of trolling, I kindly ask you to leave.

**Comment by itslukej** [comment]
> 3. Deploys got a breach -> They did email the clients regarding this issue and recommended that all passwords should be changed.

AFAIK The email sent regarding the breach was missing some key information, notably that the attacker had access to credit card information (if available in WHMCS)
>MCSHosting is owned by NickE, who literally attacked a competitor to steal customers (and surprise surprise, that competitor was DeluxeNode which later became Deploys.io).

Unsure how this is related.
> While I do agree that Deploys have had a ton of issues within the last month, it is important to note that we only list scammers and hosts who get involved in shady/illegal acts (such as MCSHosting).

And an Illegal act isn't failing to report a breach to their local data protection authority and misinforming users about it?
> So yes, Deploys.io's service has been quite frankly bad since the outage. However, I do not see them lying/scamming/committing crimes. This is purely a case of low uptime and horrible support.

See above.

**Comment by Ethereal739** [comment]
> > 1. Deploys got a breach -> They did email the clients regarding this issue and recommended that all passwords should be changed.
> 
> AFAIK The email sent regarding the breach was missing some key information, notably that the attacker had access to credit card information (if available in WHMCS)
> 
> > MCSHosting is owned by NickE, who literally attacked a competitor to steal customers (and surprise surprise, that competitor was DeluxeNode which later became Deploys.io).
> 
> Unsure how this is related.
> 
> > While I do agree that Deploys have had a ton of issues within the last month, it is important to note that we only list scammers and hosts who get involved in shady/illegal acts (such as MCSHosting).
> 
> And an Illegal act isn't failing to report a breach to their local data protection authority and misinforming users about it?
> 
> > So yes, Deploys.io's service has been quite frankly bad since the outage. However, I do not see them lying/scamming/committing crimes. This is purely a case of low uptime and horrible support.
> 
> See above.

Someone used the age argument above so I would like to put that out there.

As for Credit Card information, they seem to be using Stripe. There is no card information stored on their system whatsoever.

**Comment by Daxorinator** [comment]
The attacker DID NOT have access to credit card information. It was not stored on WHMCS, this was clarified multiple times now.

**Comment by TheDeathStreamer** [comment]
> > > "I am great friend with deploys I have never had any trouble with him"
> > > I think this statement speaks for itself lmao...
> > > Where's this deploys guy can I meet him?
> > 
> > 
> > Dxploys lost his phone fishing and i've been talking to him on discord lately was talking to him yesterday.
> 
> Deploys is not even a person and if you are only here to for the sole purpose of trolling, I kindly ask you to leave.

Ohh nvm im talking about the wrong deploys, the one im talking about owns a vpn hosting company I only thought it was him because they have the same exact logo and kinda of the same name. Sorry again

**Comment by Daxorinator** [comment]
"VPN hosting company with same exact logo and kinda same name"

Get the fuck outta here.

**Comment by TheDeathStreamer** [comment]
> "VPN hosting company with same exact logo and kinda same name"
> 
> Get the fuck outta here.

? https://i.imgur.com/jQawUML.png litterally he owns ddosdefened, same exact logo and kinda same name I didnt know.

**Comment by Daxorinator** [comment]
That's literally a scam....

**Comment by itslukej** [comment]
This is getting off-topic quickly...

**Comment by Dot-Rar** [comment]
>As for Credit Card information, they seem to be using Stripe. There is no card information stored on their system whatsoever.

>The attacker DID NOT have access to credit card information. It was not stored on WHMCS, this was clarified multiple times now.

We've clarified that card information was not available to the attacker. However, CodeCo also clarified that the attacker did in fact gain access to the WHMCS database - other information was accessed, and is still classed as a breach and must be reported to the ICO.

**Comment by Daxorinator** [comment]
and was it reported to the ICO?

**Comment by RhvsW** [comment]
> and was it reported to the ICO?

https://www.youtube.com/watch?v=cMTAUr3Nm6I

**Comment by Dot-Rar** [comment]
>and was it reported to the ICO?


>Yeah, that's it. Didn't report to the ICO or anything.

**Comment by FrozenPvPs** [comment]
1. Dxploys is not affiliated with Deploys.io
2. I was a previous client of deploys.io. I looked through all of my emails from deploys.io and I have no email regarding a data breach. 
3. Staff that are 16-18 are perfectly fine. I even know some hosting company's that have staff that are 13 (that are actually good). As long as they have the qualifications and the proper knowledge.

**Comment by Dot-Rar** [comment]
>I was a previous client of deploys.io. I looked through all of my emails from deploys.io and I have no email regarding a data breach.
iirc it was the 26th of January

**Comment by itslukej** [comment]
> 3. Staff that are 16-18 are perfectly fine. I even know some hosting company's that have staff that are 13 (that are actually good). As long as they have the qualifications and the proper knowledge.

\> 13
\> Qualifications and proper knowledge

**Comment by TheFeldi** [comment]
traceroute sta-01.game.dc1.us.infra.deploys.io
traceroute to sta-01.game.dc1.us.infra.deploys.io (77.83.2.11), 30 hops max, 60 byte packets
 1  gateway (10.64.38.178)  0.828 ms  0.780 ms  0.754 ms
 2  10.66.0.1 (10.66.0.1)  0.733 ms  0.721 ms  0.690 ms
 3  * * *
 4  10.1.94.208 (10.1.94.208)  1.518 ms  1.551 ms 10.1.94.214 (10.1.94.214)  1.531 ms
 5  208-225-47-212.int.cloud.online.net (212.47.225.208)  1.526 ms 200-225-47-212.int.cloud.online.net (212.47.225.200)  1.478 ms 254-225-47-212.int.cloud.online.net (212.47.225.254)  1.477 ms
 6  195.154.1.188 (195.154.1.188)  1.448 ms  1.038 ms 51.158.8.181 (51.158.8.181)  0.586 ms
 7  * lag-109.ear2.Paris1.Level3.net (212.3.235.201)  6.067 ms *
 8  * * *
 9  HIVELOCITY.ear1.Dallas1.Level3.net (4.71.220.242)  112.944 ms  113.010 ms  113.005 ms
10  * * *
11  * * *
12  * * *
13  te0-0-0-6.cr01.dal.nexril.net (66.206.18.74)  111.602 ms  111.816 ms  111.799 ms
14  be2.cr01.dal.nexril.net (192.34.100.11)  111.789 ms  112.012 ms  111.806 ms
15  po1.agg01.dal.nexril.net (192.34.100.3)  110.801 ms  110.593 ms  110.550 ms
16  38.107.214.10 (38.107.214.10)  111.377 ms  111.152 ms  110.640 ms
17  77.83.2.11 (77.83.2.11)  111.447 ms  111.522 ms  111.415 ms

where is voxility

**Comment by TonyS95** [comment]
thats their dallas location they dont have voxility there yet

**Comment by TonyS95** [comment]
its hivelocity with corero but its very bad.

**Comment by Dot-Rar** [comment]
>thats their dallas location they dont have voxility there yet

Yet they advertise Voxility

**Comment by TonyS95** [comment]
Yes they do they have it for their eu location the reason they dont have it in dallas because voxility doersnt have a pop and it would add major latency

**Comment by Dot-Rar** [comment]
>Yes they do they have it for their eu location the reason they dont have it in dallas because voxility doersnt have a pop and it would add major latency

Are you okay? Voxility has PoPs in LA, VA and FL in the US. The reason they don't use Voxility in the US is because Voxility is included with their EU colocation plan, whereas to get it in the US they'd have to purchase from Voxility directly, which they don't do since it's not cost effective since they only have a single US node. Yet, they advertise Voxility for US servers, which is straight up illegal.

**Comment by TonyS95** [comment]
they do not have a pop in dallas thats why they havent set it up and no what you are saying its not truth they are using their own network hardware.

**Comment by RhvsW** [comment]
> they do not have a pop in dallas thats why they havent set it up and no what you are saying its not truth they are using their own network hardware.

They're using the colocation providers (Nexril, Active-Servers) network equipment.

**Comment by TonyS95** [comment]
pretty sure they use their own switches

**Comment by lordjbs** [comment]
this thread is a big bruh moment

**Comment by RhvsW** [comment]
> pretty sure they use their own switches

wow switching so much owned network hardware UNREAL.

**Comment by Dot-Rar** [comment]
>they do not have a pop in dallas thats why they havent set it up and no what you are saying its not truth they are using their own network hardware.

Yeah my point is that they're using Nexril's included Corero firewall for filtering instead of Voxility, like they advertise. I've been directly told (can't remember if it was Tim / Rihan / David, but one of them) that it's because it's currently way too expensive, but they'll be purchasing off Voxility directly in a few months, but as it stands at the moment, what they're doing is completely illegal.

**Comment by itslukej** [comment]
I think enough has been said now 💪 

Motion to merge

**Comment by itslukej** [comment]
Any update on this?

**Comment by hb4uhomebrew** [comment]
> Any update on this?

It's not a summerhost, the end.

**Comment by KidsAreSalty** [comment]
this should conclude it https://deploys.io/ :/

**Comment by Andre601** [comment]
> this should conclude it https://deploys.io/ :/

Not really as a changed site from __one single owner__ claiming it's over doesn't proof anything.
In fact did the main owner CodeCo say [that they don't stop](https://user-images.githubusercontent.com/11576465/66333142-9511fa80-e936-11e9-9592-dde05dc61109.png).

**Comment by Daxorinator** [comment]
2 ex-management staff actually, between the 3 of them they were supposed to have equal ownership.

**Comment by DavidRazdolski** [comment]
@Andre601 - Hey, it's the OP of the post on deploys.io (former management). The fact rihan addressed none of the points that were made and instead just hid and rebranded to avoid negative PR speaks miles on its own. There is more than sufficient reason to add them to the list. +1 on the merge

**Comment by Andre601** [comment]
> 2 ex-management staff actually, between the 3 of them they were supposed to have equal ownership.

One of the did to my knowledge terminate their account completely and went away without notice.


> @Andre601 - Hey, it's the OP of the post on deploys.io (former management). The fact rihan addressed none of the points that were made and instead just hid and rebranded to avoid negative PR speaks miles on its own. There is more than sufficient reason to add them to the list. +1 on the merge

From what I found out is the reason not just to "avoid negative PR speaks" but more the fact that he (no longer) has access to site domain (since you changed the site, I wouldn't be surprised if you caused this in any way.)

**Comment by Daxorinator** [comment]
First point you made is complete bullshit, nobody "terminated their account completely and went away without notice"

The domain never belonged to Deploys.io Holdings Ltd.

**Comment by Andre601** [comment]
> First point you made is complete bullshit, nobody "terminated their account completely and went away without notice"
> 
> The domain never belonged to Deploys.io Holdings Ltd.

Then give actual valid proof instead of just claiming it.

**Comment by Daxorinator** [comment]
What proof do you want...? Which ex-member of staff "terminated their account completely and went away without notice".

Possible options are:
Rihan,
David,
Tim.

Do you even know which one your claiming left?

The domain CLEARLY never belonged to Deploys.io Holdings Ltd otherwise David wouldn't have taken it back under his control. David himself can confirm that on this PR, he was allowing Rihan to use the Deploys brand including the domain, but ultimately that all belongs to him and seizing it back is fully within his rights.

**Comment by Alontrle** [comment]
Also, CodeCo has been ignoring every message of people trying to reach out, and has done nothing to resolve this situation. 

Here is David saying CodeCo bought a phone with deploys money https://cdn.discordapp.com/attachments/388314926655078410/630786064235364369/unknown.png

Here is David saying CodeCo owes him $2,500 https://media.discordapp.net/attachments/388314926655078410/630786314198843413/unknown.png

Here is CodeCo clearly not responding to the hundreds of people asking him for info every day. http://prntscr.com/pgjga0

Here is David saying he owns the Deploys.io domain and branding http://prntscr.com/pgjhjj

Here is David saying that deploys did not OWN their hardware, but had LEASE TO OWN contracts with a company that went bunk. http://prntscr.com/pgji5x

Here are some bad decisions made by the immature owner http://prntscr.com/pgjis5

Here is people saying that he was giving extra perks in exchange for good reviews, which extras were also removed in 2 weeks after being added. http://prntscr.com/pgjjpo http://prntscr.com/pgjjbq

Here is people saying that CodeCo's mother is owner of the company http://prntscr.com/pgjkov

I have no proof any of these claims are correct but a lot of information is being sent around in the discord so I figured we can bring it here, a much easier way to manage and view the information.

**Comment by Ethereal739** [comment]
> Also, CodeCo has been ignoring every message of people trying to reach out, and has done nothing to resolve this situation.
> 
> Here is David saying CodeCo bought a phone with deploys money https://cdn.discordapp.com/attachments/388314926655078410/630786064235364369/unknown.png
> 
> Here is David saying CodeCo owes him $2,500 https://media.discordapp.net/attachments/388314926655078410/630786314198843413/unknown.png
> 
> Here is CodeCo clearly not responding to the hundreds of people asking him for info every day. http://prntscr.com/pgjga0
> 
> Here is David saying he owns the Deploys.io domain and branding http://prntscr.com/pgjhjj
> 
> Here is David saying that deploys did not OWN their hardware, but had LEASE TO OWN contracts with a company that went bunk. http://prntscr.com/pgji5x
> 
> Here are some bad decisions made by the immature owner http://prntscr.com/pgjis5
> 
> Here is people saying that he was giving extra perks in exchange for good reviews, which extras were also removed in 2 weeks after being added. http://prntscr.com/pgjjpo http://prntscr.com/pgjjbq
> 
> Here is people saying that CodeCo's mother is owner of the company http://prntscr.com/pgjkov
> 
> I have no proof any of these claims are correct but a lot of information is being sent around in the discord so I figured we can bring it here, a much easier way to manage and view the information.

His mom's name is on the company record so there's that

**Comment by Alontrle** [comment]
They advertised SSD hosting, but actually used HDDs. We TorchNode LLC experienced this and when we contacted them we were told it was a temporary solution for 2-3 days while they moved to a new supplier. This transfer did happen, but it was only an in-house downgrade within their existing supplier an all they did was move to a slower CPU with the same crappy HDDs.

http://prntscr.com/pgk7mx

**Comment by Alontrle** [comment]
Also I'd like to note, we were promised compensation, which we never received and our ticket sat open for 3+ days with no response. They showed NO intentions of refunding affected clients. I heard very very few people got refunds. I cannot prove this as they have closed the ticket and I was banned from the discord. I was later un-banned but I had no purpose to stay,

**Comment by Ethereal739** [comment]
> They advertised SSD hosting, but actually used HDDs. We TorchNode LLC experienced this and when we contacted them we were told it was a temporary solution for 2-3 days while they moved to a new supplier. This transfer did happen, but it was only an in-house downgrade within their existing supplier an all they did was move to a slower CPU with the same crappy HDDs.
> 
> http://prntscr.com/pgk7mx

Do you have screenshots of the HDD conversation?

**Comment by Alontrle** [comment]
> > They advertised SSD hosting, but actually used HDDs. We TorchNode LLC experienced this and when we contacted them we were told it was a temporary solution for 2-3 days while they moved to a new supplier. This transfer did happen, but it was only an in-house downgrade within their existing supplier an all they did was move to a slower CPU with the same crappy HDDs.
> > http://prntscr.com/pgk7mx
> 
> Do you have screenshots of the HDD conversation?

No sadly, as stated before I was banned from their discord and all my tickets closed before I could get screenshots. David was the one who banned me, not CodeCo though but I was later un-banned. Hard to get proof when they closed all your tickets without ever actually completing them.

**Comment by ThePixelord** [comment]
Shouldn't these be moved to the `Frozen Winter Hoaxes` section now?
Their site's now [RIP](https://deploys.io/), but I saved a copy of the current page on [the wonders of technology](http://web.archive.org/web/20191011205626/https://deploys.io/) (just out of boredom).

**Comment by Ethereal739** [comment]
I will add it to Frozen winter host today.

**Comment by RhvsW** [comment]
> I will add it to Frozen winter host today.

It still exists under a new brand with the same company?

**Comment by Ethereal739** [comment]
> > I will add it to Frozen winter host today.
> 
> It still exists under a new brand with the same company?

To be frank, Rihan has not been able to say anything for himself ever since David took over the domain and he has not been online nor responding to messages in the last few days. I think it is over for Deploys.io.

**Comment by ThePixelord** [comment]
So, to clarify, deploys's going in the list although (I'm *guessing*) with a link to/mention of dashflo?

**Comment by Ethereal739** [comment]
Yes

**Comment by hb4uhomebrew** [comment]
🦀🦀🦀🦀 DEPLOYS IS OVER 🦀🦀🦀🦀

**Comment by DefinitelyNotJosh** [comment]
>  DEPLOYS IS OVER

Well done! How did you figure that one out, Sherlock?

**Comment by ThePixelord** [comment]
> > DEPLOYS IS OVER
> 
> Well done! How did you figure that one out, Sherlock?

doesn't matter, *crab rave time boys 🦀🦀🦀🦀🦀*

**Comment by hb4uhomebrew** [comment]
> > > DEPLOYS IS OVER
> > 
> > 
> > Well done! How did you figure that one out, Sherlock?
> 
> doesn't matter, _crab rave time boys 🦀🦀🦀🦀🦀_
🦀🦀🦀🦀🦀🦀🦀🦀🦀🦀🦀🦀🦀🦀🦀🦀🦀🦀🦀🦀🦀🦀🦀🦀🦀

**Comment by DavidRazdolski** [comment]
To anyone with a copy of the original email still in their inboxes, please shoot me a DM on discord (david#0101) [@Dot-Rar]. Needed for various reasons (including ICO/GDPR violations)

@Ethereal739 still needs to be PR'd, btw

File: index.php [hunk 1]
```diff
@@ -294,6 +294,9 @@
 			    </li>
 			    <li data-placement="left" data-toggle="tooltip" title="A scummy host selling very sketchy WHMCS Licenses. Failing to deliver a product and ended up suspending the client's friend (who had nothing to do with this) upon getting a chargeback. They also offer free additional resources for raiding competitors' TrustPilot page. Avoid at all cost. ">
                             <a href="https://mcfh.ltd">MCFreeHosting LTD</a>
+                        </li>
+                        <li data-placement="left" data-toggle="tooltip" title="Terrible uptime & lack of transparency with customers. Managed by 15-16 year olds.">
+                            <a href="https://dashflo.net/">Dashflo.net</a> (formerly <a href="https://deploys.io">Deploys.io</a>)
```
Comment (nikrozman21) [inline index.php]: Needs a fix here. There's a ) sticking out.

**Pull request merged**